lib/presentation/pages/home_page.dart

```diff
@@ -40,6 +40,9 @@ class _HomePageState extends State<HomePage> {
       body: PageView(
         onPageChanged: (index) {
           setState(() {
+            if (index == 0){
```

Comment:
```suggestion
            if (index == 0) {
```

Reply:
We have analyzer to take care of those kinds of organization.

If this issue is not coming from there, we should not fix it individually